### PR TITLE
droppdeadz/#6 politician profile component

### PR DESCRIPTION
**Pull request by droppdeadz**
Finished the politician profile component krub. Please review :D

### CODE REVIEW

File: src/components/PoliticianProfile/PoliticianProfile.svelte [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+				</div>
+			</div>
+			<div>
+				<p class="text-lg leading-[28px] text-text-01 font-bold font-serif">{fullname}</p>
+				<p class="leading-[18px] text-text-02">พรรค{party.name}</p>
+				<p class="leading-[18px] text-text-02">{role}</p>
+			</div>
+		</div>
+	</div>
+{:else}
```
Comment (Th1nkK1D) [inline src/components/PoliticianProfile/PoliticianProfile.svelte:49]: Small and large sizes should have the same HTML structure, can we refactor it to use the same HTML template but a different "style" or "property" instead? That way, we will have less duplicated code between small and large sizes.

**Review by Th1nkK1D — CHANGES_REQUESTED** [review]
Thank you! The code is working as expected. I have 2 feedbacks.

File: src/components/PoliticianProfile/PoliticianProfile.svelte [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	$: fullname = `${firstname} ${lastname}`;
+</script>
+
+{#if isLarge}
+	<div class="p-2 font-sans">
+		<div class="flex gap-4">
+			<div class="w-[64px] h-[64px] relative">
+				<img
+					src={avatar}
+					alt={`${fullname}-${id}`}
```
[the remark this answers is not in the document]
Comment (droppdeadz) [reply src/components/PoliticianProfile/PoliticianProfile.svelte]: Thank you krub. It's fixed 🙏

File: src/components/PoliticianProfile/PoliticianProfile.svelte (hunk 1 above)
Comment (droppdeadz) [reply src/components/PoliticianProfile/PoliticianProfile.svelte]: Fixed krub 🙏

**Review by Th1nkK1D — CHANGES_REQUESTED** [review]
Hi! Your code is now looking great. I have one last request for the story. Can you split the `isLarge` variation into another variant in `Hst.Variant` to make it more visible to a developer who will use it?
- [Histoire doc](https://histoire.dev/guide/svelte3/stories.html#variants)
- [Example](src/components/VotingResultTag/VotingResultTag.story.svelte) from `VotingResultTag`

**Comment by droppdeadz** [comment]
> Hi! Your code is now looking great. I have one last request for the story. Can you split the `isLarge` variation into another variant in `Hst.Variant` to make it more visible to a developer who will use it?
> 
> * [Histoire doc](https://histoire.dev/guide/svelte3/stories.html#variants)
> * [Example](src/components/VotingResultTag/VotingResultTag.story.svelte) from `VotingResultTag`

Done krub

**Review by Th1nkK1D — APPROVED** [review]
LGTM now! Thank you for your contribution. We will contact you back at the end of this month for the stickers.

**Comment by Th1nkK1D** [comment]
Thank you for the Hacktoberfest participation! Please fill up the [form](https://airtable.com/appgGYIo458vDkrQN/shrZMe2hZBa6tQXvz) before 6 November 2023 to receive our limited-edition stickers.

**Pull request merged**